src/image_builder.py

```diff
@@ -80,6 +81,14 @@ def image_builder(buildspec, image_types=[], device_types=[]):
     :param image_types: <list> list of image types
     :param device_types: <list> list of image device type
     """
+    with open(buildspec, "r") as bf:
+        pointer_check = yaml.safe_load(bf)
+        pointer = pointer_check.get("buildspec_pointer")
+        if pointer:
+            FORMATTER.print(f"Buildspec {buildspec} points to another buildspec file {pointer}")
```

Comment:
check for build context if this is a pointer. In PRs this is ok, elsewhere we will want a failure